lib/spack/spack/test/main.py

```diff
@@ -23,6 +26,8 @@ def test_get_version_no_match_git(tmpdir, working_env):
     assert spack.spack_version == get_version()
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
```

Comment:
This also looks like another simple case where I assume the functionality works and the test fails because we generate a mock exe as a bash script, which wouldn't run there (I think it's fine to leave this as a skip but the point being, where there the explanation is simple I think it would be helpful to include in the `reason` or an adjacent comment).

Reply:
Addressed.